### DISCUSSION TITLE
Discussions are underused. Add a direct link to unanswered discussions

### DISCUSSION THREAD

**Discussion by annaesvensson** [community/community#68976] [Discussions]
### Select Topic Area

Product Feedback

### Body

In my opinion, community discussions are a great feature, but new users often only ask questions and don't participate in answering question. The more people are active in community discussions, the better it would be for everyone. How about adding a link to `Unanswered discussions` as a call to action? See screenshot below:

![Screen Shot 2023-10-02 at 10 08 29](https://github.com/community/community/assets/14218799/034cd5b3-e4f1-41d2-b88e-ec929e2c1b90)

**Comment by Talhadmr** [comment]
I agree with you on this matter. A great potential is being wasted here.

**Comment by shortcut119** [comment]
I completely agree. a more visible way to access unanswered discussions could really strengthen the community aspect of GitHub.

Many contributors who want to help tend to look for open questions, but the current navigation flow hides those behind several filters. A dedicated “Unanswered” link or tab near the Discussions menu (similar to “Issues → Open”) would make it easier for mentors, maintainers, and active helpers to jump straight into topics that still need attention.

A few simple UI or workflow improvements that might encourage engagement:

Add a visible “Unanswered” shortcut at the top of each category view.

Highlight posts without accepted answers in a subtle color or icon, to catch contributors’ eyes.

Introduce a “Help Needed” badge automatically applied to unanswered questions after a certain time (for example, 48 hours).

Provide analytics for maintainers showing which categories have the most unanswered posts — so they can direct community energy there.

These small UX touches could make Discussions feel more dynamic and balanced, where contributors can immediately see where help is most needed.

It might even increase the acceptance rate of useful answers — a win for both the community and the platform.

**Comment by ab9717208-bisht** [comment]
how much storage requirement for run hippius subnets
